cmd/executor/cmd/root.go

```diff
@@ -110,21 +112,45 @@ func checkDockerfilePath() error {
 	return errors.New("please provide a valid path to a Dockerfile within the build context")
 }
 
-// resolveSourceContext unpacks the source context if it is a tar in a GCS bucket
+// resolveSourceContext unpacks the source context if it is a tar in a bucket
 // it resets srcContext to be the path to the unpacked build context within the image
 func resolveSourceContext() error {
 	if srcContext == "" && bucket == "" {
```

Comment:
We need to decide what flag to use here - or introduce a more generic wording here.

Reply:
I think we should keep both flags so that this change is backwards compatible. Let's make sure that either 
`--context` or `--bucket` is specified (and return an error if both or neither are), and pass whichever one is specified into `GetBuildContext()` (mentioned below)

Also, if we pass in `--bucket` with no prefix, assume a GCS bucket so that it's still backwards compatible :)